models/topics.models.js

```diff
@@ -7,4 +8,13 @@ exports.selectAllTopics = () => {
         }
         return rows;
     })
+}
+
+exports.selectAllEndpoints = () => {
```

Comment:
Models are reserved for database queries only, you should only use a controller for this endpoint :)